### ISSUE TITLE
Tests\Feature\Api\PostEntriesTest::testPostEntries ["filtering 'expense:false'"] failed

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/money-tracker#44]
This [test](https://github.com/jdenoc/money-tracker/blob/455613cca644e93e5b95346f0e233dfcc09c30d0/tests/Feature/Api/PostEntriesTest.php#L100-L125) failed with this error:

```
1) Tests\Feature\Api\PostEntriesTest::testPostEntries with data set "filtering 'expense:false'" (array(false))
Expected status code 200 but received 404.
Failed asserting that false is true.
/home/travis/build/jdenoc/money-tracker/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestResponse.php:79
/home/travis/build/jdenoc/money-tracker/tests/Feature/Api/PostEntriesTest.php:120
```

This is the failed [build](https://travis-ci.org/jdenoc/money-tracker/builds/302005311).

**Comment by jdenoc** [comment]
Created #50 to try and identify the cause/problem.

**Comment by jdenoc** [comment]
Fixed by #62 & #69
